roles/online_docs/templates/mkdocs/docs/apptainer.md

```diff
@@ -0,0 +1,595 @@
+- [Other considerations](#other-considerations)
+    - [Architecture considerations](#architecture-considerations)
+    - [Licence considerations](#licence-considerations)
+- [Environment variables](#environment-variables)
+- [Debug information](#debug-information)
+- [Additional examples](#additional-examples)
+    - [Alpine as regular user build static file via sandbox directory](#alpine-as-regular-user-build-static-file-via-sandbox-directory)
+    - [Simple CentOS 7 example via sandbox](#simple-centos-7-example-via-sandbox)
+
+## About Apptainer
+
+### In short
+
+> Container is an isolated environment that holds the software and its dependencies and configurations. Containers can be run on any machine with compatible container technology.
+Container instance can be run (deployed) from an existing image. Therefore users must first either build (create) or copy an existing image (f.e. pull from a public registry). They can also share their prebuilt images with each other, and running containers from those, they can expect always to get the same results, regardless of the system on which the containers are run.
+> Apptainer is one of the container technology implementations; Docker is, for example, another. They work differently manner and are not fully compatible.  Apptainer is suited for HPC environments but can also run images from the Docker repository.
+> Users can deploy containers, and inside those containers can access the same resources as they can access outside of those containers - nothing more.
+> Container technology described here is focused only on building and deploying Linux containers on Linux host systems.
+
+Documentation is available on the official [Apptainer's documentation website](https://apptainer.org/docs/)
+
+### Containers and apptainer
+
+About Linux containers
+* are a method for running multiple isolated Linux systems (containers) on a control host using a single Linux kernel,
+* provides the cgroups functionality that allows limitation and prioritization of resources (CPU, memory, block I/O, network, etc.) without the need for starting any virtual machines.
+
+Apptainer is a container platform
+* highly optimized to be run on a laptop or high-performance cluster,
+* extremely fast and lightweight - as there is no virtualization, and images are simply executed just as any other program,
+* enables a secure way of running containers - images can be easily verified,
+* limits security risk, as it can only access user-accessible resources,
+* can directly run images from the Docker repository (via path `docker://...`).
+
+Apptainer container image
+* is a file (f.e. `ubuntu_18.04.sif`) containing all programs and libraries needed to execute that environment,
+* can contain a small subset of programs (with libraries), or entire Linux operating system (like Alpine, CentOS or Ubuntu),
+* is highly portable between various Linux operating systems and environments - it only requires installed Apptainer,
```

Comment:
Maybe add a link to the limitations. You will need Apptainer installed and a compatible CPU architecture ;)